cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -55,10 +60,18 @@ func buildClusterManagerOpts(t timeoutOptions) ([]clustermanager.ClusterManagerO
 	}, nil
 }
 
+func markFlagHidden(cmd *cobra.Command, flagName string) {
+	if err := cmd.Flags().MarkHidden(flagName); err != nil {
+		logger.V(5).Info("Warning: Failed to mark flag as hidden: " + flagName)
```

Comment:
This seems wrong. Typically this will only error if the flag doesn't exist so warning on it strikes me as a significant issue, possibly a cause for a panic given the end-user can't do anything about it.

Reply:
markFlagHidden was merged in a separate PR (https://github.com/aws/eks-anywhere/pull/2959/), think git diff picked it up since I moved my functions underneath it. Is that something that I should address in this PR? Or should I make sure that the diff doesn't regard this as a change?

Reply:
If it has nothing to do with your PR then I would recommend rebasing your changes on-top of main so we can see its got nothing to do with this.

Reply:
Rebased changes and backed out of flag changes. This should no longer show up as modified